pom.xml

```diff
@@ -2645,7 +2647,18 @@
       <properties>
         <scala.version>2.11.8</scala.version>
         <scala.binary.version>2.11</scala.binary.version>
+        <jline.version>2.12.1</jline.version>
+        <jline.groupid>jline</jline.groupid>
```

Comment:
It sounds like this property is no longer even needed? I don't see that it's modified anywhere from "jline"